openedx/core/djangoapps/user_api/urls.py

```diff
@@ -15,4 +16,19 @@
         AccountView.as_view(),
         name="accounts_api"
     ),
+    url(
+        r'^v0/preferences/' + USERNAME_PATTERN + '$',
+        PreferencesView.as_view(),
+        name="preferences_api"
+    ),
+    url(
```

Comment:
Thanks. I cherry-picked in the wrong order and that seemed to cause some strange merge issues like this.